src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -14,7 +14,11 @@
      * то возвращать 1
      * если не угадал ничего - возвращать 0.</p>
      */
-    public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+    public int matchResult(int result1, int result2, int bet1, int bet2)
+    {
+        int general_result = result1==bet1&&result2==bet2?2:
```

Comment:
Code Style:
-naming,
-whitespaces.
Please check this link: https://mate-academy.github.io/style-guides/java/java.html